Api/app/models/request.rb

```diff
@@ -15,10 +17,13 @@ class Request < ApplicationRecord
     rejected: "Rejected",
```

Comment:
Can you tell me `rejected,` and `cancel`  objects

Reply:
It's a little mistake but it doesn't affect anything else because I don't call the variable.

Reply:
Ok, you should fix it in the next PR, thank you